libqtile/scripts/migrate.py

```diff
@@ -260,4 +252,35 @@ def add_subcommand(subparsers, parents):
         action="store_true",
         help="Automatically apply diffs with no confirmation.",
     )
-    parser.set_defaults(func=do_migrate)
+    parser.add_argument(
+        "--show-diff", action="store_true", help="When used with --yes, will still output diff."
```

Comment:
is there a --yes arg?

Reply:
Yes. It was there before so it may not be visible in a truncated diff.